plugins/kubernetes/app/models/watchers/deploy_watcher.rb

```diff
@@ -0,0 +1,83 @@
+require 'celluloid'
+require 'celluloid/autostart'
+
+module Watchers
+  # Instantiated when a Kubernetes deploy is created to watch the status
+  class DeployWatcher
+    include Celluloid
+    include Celluloid::Notifications
+
+    finalizer :on_termination
+
+    def initialize(release)
+      @release = release
+      @current_rcs = {}
+      Rails.logger.info "Start watching K8s deploy: #{@release}"
```

Comment:
I'd say either put everything in the rails logger, or keep the k8s log file, but not 3 :)